Formula/c14-cli.rb

```diff
@@ -0,0 +1,22 @@
+class C14Cli < Formula
+  desc "Manage your Online C14 archives from the command-line"
+  homepage "https://github.com/online-net/c14-cli"
+  url "https://github.com/online-net/c14-cli/archive/0.1.tar.gz"
+  sha256 "cff3597273daff87e8d6e85cfef2b4d83400f0a0a905f39a4a67560b4966513c"
+  head "https://github.com/online-net/c14-cli.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GOBIN"] = buildpath
+    (buildpath/"src/github.com/online-net/c14-cli").install Dir["*"]
+
+    system "go", "build", "-o", "#{bin}/c14", "-v", "-ldflags", "-X  github.com/online-net/c14-cli/pkg/version.GITCOMMIT=homebrew", "github.com/online-net/c14-cli/cmd/c14/"
```

Comment:
[This kind of style](https://github.com/Homebrew/homebrew-core/blob/0c14c6c7685c44e8771b1d3f2732cccb66dcfd75/Formula/wellington.rb#L37-L39) is a nicer format to read than one extremely long line FWIW.